data/gamerules.txt

```diff
@@ -45,7 +45,43 @@ gamerules
 	# ALLOWABLE VALUES: any integer >= 0
 	"npc max mining time" 3600
 	
-	# Ships with the "frugal" personality will only use weapons that consume ammo or fuel while their health is at or below this fraction.
+	# Ships with the "frugal" personality will only use weapons that consume ammo or fuel while their
+	# health is at or below this fraction.
 	# DEFAULT: .75 (75 percent)
 	# ALLOWABLE VALUES: any value between 0. and 1.
-	"universal frugal threshold" .75
+	"universal frugal threshold" 0.75
+
```

Comment:
I think this makes for a much more useful comment than what we have going on further down.
```suggestion

	# Depreciation related game rules:

	# The depreciated value of an item that is age days old is calculated with the following equation:
	# Where: "base value" is the undepreciated value of the item;
	# min is "depreciation min";
	# "grace period" is "depreciation grace period";
	# daily is "depreciation daily"; and
	# "max age" is "depreciation max age", and
	# age is at greater than "grace period" and less than "max age".
	# "depreciated value" = "base value" *
	#   (min + (1 - min) * (daily ^ (age - "grace period")) * ("max age" + "grace period" - age) / "max age")

```